tee-worker/scripts/litentry/generate_release_pkg.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euo pipefail
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+WORKER_DIR="${ROOT_DIR}/tee-worker"
+
+
+# Edit the following configs:
+COMMERCIAL_KEY_PATH=${WORKER_DIR}/enclave-runtime/Enclave_private.pem
```

Comment:
This should come from env or GHA secret I think

Reply:
This key is not yet clear. So I put the key for development here. 
Yes, agree. It should be some env or GHA secret.